common/plugins/eu.esdihumboldt.hale.common.core/src/eu/esdihumboldt/hale/common/core/report/impl/AllInMemoryReporter.java

```diff
@@ -0,0 +1,176 @@
+/*
+ * Copyright (c) 2012 Data Harmonisation Panel
```

Comment:
Severe case of outdated header :)

Reply:
Yes, but that's ok because that's actually the original `DefaultReporter` class, just renamed.